packages/lesswrong/components/messaging/MessageItem.tsx

```diff
@@ -48,11 +50,18 @@ const styles = (theme: ThemeType): JssStyles => ({
   },
   messageBody: {
     '& a': {
-      color: theme.palette.primary.light
+      color: theme.palette.primary.light,
+      wordWrap: "break-word",
+      overflowWrap: "break-word",
+      whiteSpace: "normal",
     },
     '& img': {
       maxWidth: '100%',
     },
+    // Workaround to make sure links don't overflow the message box
+    '& .LWTooltip-root': {
+      display: 'inline',
+    },
```

Comment:
All the changes here are essentially bug fixes to make messages render more nicely on smaller screens (e.g. links didn't wrap properly before)